daiquiri/__init__.py

```diff
@@ -105,6 +106,21 @@ def logging_excepthook(exc_type, value, tb):
         logging.captureWarnings(True)
 
 
+def setup_colors(palette=formatter.Swatches.Default):
```

Comment:
It's a refinement of adding a `palette` kwarg to `setup`. Especially since colors don't have to be configured every time or even at all.